class.jetpack-options.php

```diff
@@ -404,4 +414,27 @@ static function get_raw_option( $name, $default = null ) {
 		return $value;
 	}
 
+	/**
+	 * This function lets us by pass certain options to be gotten via the raw sql calls.
+	 * Instead we should use the regular wp calls.
+	 * @param $name Option name
+	 *
+	 * @return bool
+	 */
+	static function by_pass_raw_option( $name ) {
+
+		if ( Jetpack_Constants::get_constant( 'JETPACK_DISABLE_RAW_OPTION' ) ) {
+			return true;
+		}
+		/**
+		 * Allows us to disable the some raw options.
```

Comment:
` Allows us to disable the some raw options.` =>  `Allows to disable particular raw options.`